mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -562,6 +618,43 @@ class DistributedLDAModel private[ml] (
    */
   @Since("1.6.0")
   lazy val logPrior: Double = oldDistributedModel.logPrior
+
+  @Since("1.6.0")
+  override def write: MLWriter = new DistributedLDAModel.DistributedLDAModelWriter(this)
+}
+
+
+@Since("1.6.0")
+object DistributedLDAModel extends MLReadable[DistributedLDAModel] {
+
+  private[DistributedLDAModel]
+  class DistributedLDAModelWriter(instance: DistributedLDAModel) extends MLWriter {
+
+    override protected def saveImpl(path: String): Unit = {
+      DefaultParamsWriter.saveMetadata(instance, path, sc)
+      instance.oldDistributedModel.save(sc, path)
```

Comment:
This may not work; it will attempt and fail to overwrite the metadata saved on the previous line.  Maybe put it in a subdirectory.